lib/internal/test_runner/test.js

```diff
@@ -129,6 +129,13 @@ function lazyAssertObject() {
     for (let i = 0; i < methodsToCopy.length; i++) {
       assertObj.set(methodsToCopy[i], assert[methodsToCopy[i]]);
     }
+
+    const { getOptionValue } = require('internal/options');
+    if (getOptionValue('--experimental-test-snapshots')) {
```

Comment:
why isnt this also part of `parseCommandLine`?

Reply:
Just because this is a temporary flag. I figured it would be simpler to remove in the future.